php/media/class-global-transformations.php

```diff
@@ -563,6 +562,29 @@ public function save_overwrite_transformations_featured_image( $post_id ) {
 		}
 	}
 
+	/**
+	 * Get the current post.
+	 *
+	 * @return WP_Post|null
+	 */
+	protected function get_current_post() {
+		/**
+		 * Filter the post ID.
+		 *
+		 * @hook    cloudinary_post_id
+		 * @default null
+		 *
+		 * @return  {WP_Post|null}
+		 */
+		$post_id = apply_filters( 'cloudinary_post_id', null );
+
+		if ( is_null( $post_id ) ) {
```

Comment:
@pereirinha I think get_post returns `null` if outside the loop and no id is provided, making this irrelevant.
However, I'm not concerned and will approve. But if you can, give it a check